UI/window-basic-main.cpp

```diff
@@ -3805,6 +3809,53 @@ QMenu *OBSBasic::AddScaleFilteringMenu(obs_sceneitem_t *item)
 	return menu;
 }
 
+QMenu *OBSBasic::AddBackgroundColorMenu(obs_sceneitem_t *item)
+{
+	QMenu *menu = new QMenu(QTStr("ChangeBG"));
+	QAction *action;
+
+	obs_data_t *privData = obs_sceneitem_get_private_settings(item);
+	obs_data_release(privData);
+
+	obs_data_set_default_int(privData, "color-preset", 0);
+	int preset = obs_data_get_int(privData, "color-preset");
+
+	action = menu->addAction("Clear", this, \
```

Comment:
Is this syntax required?  I don't understand the need for the backslash here.  See also below.